src/environments/environment.local.ts

```diff
@@ -5,7 +5,7 @@ export const environment = {
   API_WEB: '//dev.orcid.org/',
   BASE_URL: '//dev.orcid.org/',
   BLOG_NEWS: 'https://localhost:8888/about/news',
-  INFO_SITE: 'https://info.orcid.org/',
```

Comment:
let's make it work for the other environments as well

Reply:
Ok